images/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:stable-20190610 as java8-builder
+
+RUN apt update -qq && \
+    apt install wget -y
+
+RUN wget https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz -P /srv && \
+    mkdir /srv/jdk && \
+    tar -xvvf /srv/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz --strip-components=1 -C /srv/jdk
+
+FROM debian:stretch-20190610-slim
```

Comment:
sometimes customers ask for images being tagged with content sha like [here](https://github.com/mesosphere/jupyter-service/blob/master/docker/cpu/notebook/Dockerfile#L1-L3)

Reply:
added! thanks